package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-edx",
+  "version": "0.0.1",
+  "description": "ESLint config for edX JavaScript code.",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
```

Comment:
Not 100% sure I understand you here - you're saying we should test our lint ruleset by providing samples of code that should fail, and testing that they do? [ESLint already does that when they ship, though](https://travis-ci.org/eslint/eslint/jobs/139022157).

I guess beyond the correctness of ESLint rules in actually doing what they say they do, we could also get into a situation where we create a contradiction (eg, object properties have to be in kebab case, but you can't quote them). I don't know if unit testing helps us catch that though, just running it on our real codebase should be enough.